precice-config.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0"?>
+
+<precice-configuration>
+
+  <log enabled="1">
+    <sink filter="%Severity% > debug" />
+  </log>
+  
+  <solver-interface dimensions="2">
+    
+    <!-- Data fields that are exchanged between the solvers -->
+    <data:scalar name="Pressure"/>
+    <data:scalar name="CrossSectionLength"/>
+
+    <!-- A common mesh that uses these data fields -->
+    <mesh name="Fluid-Nodes">
+        <use-data name="CrossSectionLength"/>
+        <use-data name="Pressure"/>
+    </mesh>
+
+    <mesh name="Solid-Nodes">
+       <use-data name="CrossSectionLength"/>   
+       <use-data name="Pressure"/>
+    </mesh>
+
+    <!-- Represents each solver using preCICE. In a coupled simulation, two participants have to be
+         defined. The name of the participant has to match the name given on construction of the
+         precice::SolverInterface object used by the participant. -->
+    
+    <participant name="FLUID">
+      <!-- Makes the named mesh available to the participant. Mesh is provided by the solver directly. -->
+      <use-mesh name="Fluid-Nodes" provide="yes"/>
+      <use-mesh name="Solid-Nodes" from="Solid"/>
+      <!-- Define input/output of the solver.  -->
+      <write-data name="Pressure" mesh="Fluid-Nodes"/>
+      <read-data  name="CrossSectionLength" mesh="Fluid-Nodes"/>
+      <mapping:nearest-neighbor direction="read" from="Solid-Nodes" to="Fluid-Nodes" constraint="consistent" timing="initial"/>
+      <watch-point mesh="Fluid-Nodes" name="outlet" coordinate="10.0; 0.0"/>
+    </participant>
+    
+    <participant name="Solid">
+      <use-mesh name="Solid-Nodes" provide="yes"/>
+      <use-mesh name="Fluid-Nodes" from="FLUID"/>
+      <write-data name="CrossSectionLength" mesh="Solid-Nodes"/>
+      <read-data  name="Pressure" mesh="Solid-Nodes"/>
+      <mapping:nearest-neighbor direction="read" from="Fluid-Nodes" to="Solid-Nodes" constraint="consistent" timing="initial"/>
+    </participant>
+
+    <!-- Communication method, use TCP sockets, Change network to "ib0" on SuperMUC -->
+    <m2n:sockets from="FLUID" to="Solid"/>
+
+    <coupling-scheme:serial-implicit>
+      <participants first="FLUID" second="Solid"/>
+      <max-time value="0.99"/>
```

Comment:
Why not a perfect `1`? Looks suspicious...

Reply:
Changed! That was necessary at some point but not anymore:)